### DISCUSSION TITLE
Cannot use cacheStrategy because of 'Never' type

### DISCUSSION THREAD

**Discussion by hamitgulhan** [prisma/prisma#22489] [Q&A] [answered]
### Question

I want to query like this: 
`await this.prismaService.site.findFirst({
            where: {
              url: siteDomain,
              seller_name: '',
            },
            cacheStrategy: { ttl: 60 },
          });`
But I get this error:  TS2322: Type '{ ttl: number; }' is not assignable to type 'never'.

What am I doing wrong?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @hamitgulhan 👋 

Thank you for raising this question.

How are you instantiating your Prisma Client with Accelerate? 

Can you rewrite the code as:
```js
await this.prismaService.$extends(withAccelerate()).site.findFirst({ where: { url: siteDomain, seller_name: '', }, cacheStrategy: { ttl: 60 }, })
```

**Reply by hamitgulhan** [reply]
Thank you, I was not using accelerate
